capysvc/service_definition.go

```diff
@@ -1154,3 +1157,98 @@ func (o *Operation) newFilesystemInputRemoveOperation() (*operations.FilesystemI
 		Params: &operations.FilesystemInputRemoveOperationParams{},
 	}, nil
 }
+
+func (o *Operation) newCommandExecOperation(ctx Context) (*operations.CommandExecOperation, error) {
+	parameterLoaderProvider, providerErr := ctx.ParameterLoaderProvider()
+	if providerErr != nil {
+		return nil, providerErr
+	}
+
+	var commandName string
+	if commandNameParameter, ok := o.Params["commandName"]; ok {
```

Comment:
Probably better and more legible to return earlier in the opposite condition like:
```go
if commandNameParameter, ok := o.Params["commandName"]; !ok {
  return nil, errors.New("failed to retrieve \"commandName\" parameter")
}
parameterLoader, loaderErr := parameterLoaderProvider.ParameterLoader(
...
```

Reply:
I actually tried this and it introduced even more complexity that is now related to the scope of the variables. So it doesn't seem to work well in this case.